perception/autoware_map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -1397,6 +1427,41 @@ bool MapBasedPredictionNode::isIntersecting(
   return intersection.has_value();
 }
 
+bool MapBasedPredictionNode::isLeft(const TrackedObject & object)
+{
+  lanelet::BasicPoint2d search_point(
+    object.kinematics.pose_with_covariance.pose.position.x,
+    object.kinematics.pose_with_covariance.pose.position.y);
+
+  // find current lane
+  const auto surrounding_lanelets = getCurrentLanelets(object);
+  if (surrounding_lanelets.empty()) {
+    return false;
+  }
+
```

Comment:
[NITS] It is reasonable for me to declare `search_point` after handling the early return.